Content.Client/Chemistry/UI/ChemMasterWindow.xaml.cs

```diff
@@ -37,6 +39,13 @@ public ChemMasterWindow()
             RobustXamlLoader.Load(this);
             IoCManager.InjectDependencies(this);
 
+            BufferSortButton.OnPressed += _ =>
+            {
+                if (_chemState != null)
+                {
+                    OnSortButton(_chemState);
+                }
+            };
```

Comment:
А здесь

```suggestion
            BufferSortButton.OnPressed += _ =>
            {
                _isSortingEnabled = !_isSortingEnabled;
                if (_chemState != null)
                {
                    UpdatePanelInfo(_chemState);
                }
            };
```

В `UpdatePanelInfo()` логику как в первом коммите, но с учётом `_isSortingEnabled`